app/views/modules/_footer.html.erb

```diff
@@ -13,16 +13,55 @@ Unless required by applicable law or agreed to in writing, software distributed
   specific language governing permissions and limitations under the License.
 ---  END LICENSE_HEADER BLOCK  ---
 %>
-   <div class="container-fluid">
-	  <footer>
-	   <ul>
-	   	 <li><%= image_tag "AvalonMediaSystem_Logo_A.png", alt: "Avalon Media System - #{t(:release_label)} #{Avalon::VERSION}" %>
-	     <li><a href="http://www.avalonmediasystem.org">Avalon Media System Project Website</a> </li>
-       <li><%= link_to "Contact Us", comments_path %></li>
-	   </ul>
-	   <div class="release">
-	   	<%=release_text%>
-	   	<%= git_commit_info(" (%s %s [%s])") if Rails.env == 'development' %>
-	   </div>
-       </footer>
+
+<div class="footer dark row-fluid">
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-4">
+        <h2>Quick Links</h2>
+        <ul>
+          <li><a href="/users/sign_in">Login</a></li>
+          <li><a href="/catalog?q=&search_field=all_fields&utf8=%E2%9C%93">Browse</a></li>
+        </ul>
+      </div>
+      <div class="col-sm-4">
+        <h2 class="hidden-xs">Policies</h2>
+        <ul>
+          <li><a href="/about_eraav">About ERA A<span class="plus">&plus;</span>V</a></li>
+          <li><a href="/policies">Policies</a></li>
+          <li><a href="/technology">Technology and Partnerships</a></li>
+        </ul>
+      </div>
+      <div class="col-sm-4">
+        <h2 class="hidden-xs">Get Help</h2>
+        <ul>
+          <li><a href="/contact">Contact Us</a></li>
+          <li><a href="/deposit">How to Deposit</a></li>
+        </ul>
+      </div>
+    </div>
+    <div class="row copyright">
+      <a href="https://library.ualberta.ca"> University of Alberta Libraries</a> |
+      University of Alberta, Edmonton, AB, Canada T6G 2R3 |
+      <a href="https://library.ualberta.ca/about-us/policies/privacy-policy">
+        Web Site Privacy Policy
+      </a>
+    </div>
+    <div class="row">
+      <a href="http://www.avalonmediasystem.org"><%= t(:powered_by_avalon) %></a>
+      <!-- Avalon Media System Release <%= Avalon::VERSION %> -->
```

Comment:
why a html comment? This still gets rendered into the HTML source

So put it somewhere better? (meta tag?)

Reply:
@murny do other project capture this info - any suggestions on the name of the meta tag?

Reply:
There is a Meta generator tag (historically it was used by HTML generators, but has since been used by the software that runs the website. For example any website using WordPress had a Meta generator tag to show the WordPress version. )

Take a peak at this issue in Jupiter that we wish to do (last comment by weiwei where I edited the solution in)